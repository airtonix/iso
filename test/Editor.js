const React = require('react')
const { create: render } = require('react-test-renderer')
const Editor = require('../src/Editor').default

describe('Editor', () => {
  test('renders', () => {
    const json = render(
      React.createElement(Editor, { value: 'hello', onChange: () => {} })
    ).toJSON()
    expect(json).toMatchSnapshot()
  })
})
