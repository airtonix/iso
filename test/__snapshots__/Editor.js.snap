// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Editor renders 1`] = `
<div
  className="sc-bdVaJa gTcADx"
>
  <div
    id="brace-editor"
    style={
      Object {
        "backgroundColor": "transparent",
        "color": "inherit",
        "fontFamily": "\\"Roboto Mono\\", monospace",
        "fontSize": 13,
        "height": "calc(100vh - 40px)",
        "width": "100%",
      }
    }
  />
</div>
`;
